src/components/Swap/Toolbar/Caption.tsx

```diff
@@ -59,24 +76,27 @@ function Caption({ icon: Icon = AlertTriangle, caption, color = 'secondary', too
   )
 }
 
-function GasEstimate({ gasUseEstimateUSD }: GasEstimateProps) {
+function GasEstimate({ gasUseEstimateUSD, trade }: GasEstimateProps) {
```

Comment:
This has become more than just the `GasEstimate`. It should be renamed. `TradeTooltip`?